app/services/container_dashboard_service.rb

```diff
@@ -160,4 +161,44 @@ def heatmaps
       }
     }
   end
+
+  def node_utilization
+    resource_ids = @ems.present? ? [@ems.id] : ManageIQ::Providers::ContainerManager.all.pluck(:id)
+    metrics = VimPerformanceDaily.find_entries({:tz => @controller.current_user.get_timezone})
+    metrics = metrics.where(:resource_type => 'ExtManagementSystem', :resource_id => resource_ids)
+    metrics = metrics.where("timestamp > ?", 30.days.ago)
```

Comment:
> (though i don't understand what the reason for splitting it).

What would be the syntax for keeping it in one line?

This causes an exception:
``` 
metrics.where(:resource_type => 'ExtManagementSystem', :resource_id => resource_ids, "timestamp > ?", 30.days.ago)
```